python/test/test_meas.py

```diff
@@ -7,21 +7,30 @@ def test_gaussian_curve():
     meas = dp.c.make_zCDP_to_approxDP(dp.m.make_gaussian(*input_space, 4.))
     profile = meas.map(d_in=1.)
     assert profile.epsilon(delta=0.) == float('inf')
-    assert profile.epsilon(delta=1e-3) == 0.6880024554878086
+    assert profile.epsilon(delta=1e-3) == 0.6880024554878085
     assert profile.epsilon(delta=1.) == 0.
+    assert profile.delta(epsilon=0.) == 0.1508457845622862
+    assert profile.delta(epsilon=0.6880024554878085) == 1e-3
 
     profile = dp.c.make_zCDP_to_approxDP(dp.m.make_gaussian(*input_space, 4.)).map(d_in=0.0)
     assert profile.epsilon(0.0) == 0.0
     with pytest.raises(Exception):
         profile.epsilon(delta=-0.0)
-
+    with pytest.raises(Exception):
+        profile.delta(epsilon=-0.0)
```

Comment:
I see that R has a negative zero that is not quite the same as zero, but does Python?

Also, preference for stronger checks with `pytest.raises`:
- It's probably a subclass of `Exception`?
- Usability: Is the message something an end user would understand?

Reply:
Negative zero is equivalent to positive zero, but can be used by some software to represent underflow of a negative number. 
It's a part of the IEEE-754 standard, so it's present in R as well (and any software that adheres to that standard, which is basically all software).